cfgov/unprocessed/js/modules/util/Behavior.js

```diff
@@ -0,0 +1,34 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var Behavior = {};
+
+/**
+ * @param {string} selector Used to query dom for elements.
+ * @param {string} event Event type to add to element.
+ * @param {function} eventHandler Callback for event.
+ */
+Behavior.create = function create( selector, event, eventHandler ) {
```

Comment:
Not really a big deal till it needs to be different, but NodeList is going to be the most flexible (or alternatively, selector + element, with a default of `document` like mentioned in https://github.com/cfpb/cfgov-refresh/pull/2115#discussion_r64073659). That way the scope isn't forced to be the whole document.